lib/content-services/src/lib/search/services/search-facet-filters.service.ts

```diff
@@ -134,6 +138,34 @@ export class SearchFacetFiltersService implements OnDestroy {
         });
     }
 
+    private parseTabbedFacetField() {
+        if (this.responseFacets) {
+            let fields = '';
+            this.responseFacets.forEach((facet) => fields = `${fields},${facet.field}`);
```

Comment:
maybe better would be reduce as you convert array to single value?

Reply:
Changed to reduce